dist/index.html

```diff
@@ -1,12 +1,14 @@
-<!-- HTML for static distribution bundle build -->
+﻿<!-- HTML for static distribution bundle build -->
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <title>Swagger UI</title>
     <link rel="stylesheet" type="text/css" href="./swagger-ui.css" />
-    <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
-    <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
+	  <link rel="stylesheet" type="text/css" href="./outSystems.css" />
```

Comment:
Shouldn't we annotate this line?

Reply:
yes yes. There will be changes in the index.html once we incorporate in our platform. So that, I didn't take care of this comments so far

Reply:
probably we won't use this css file

Reply:
thank you. done